### DISCUSSION TITLE
Could you pls add some ass functions？

### DISCUSSION THREAD

**Discussion by yelang5460** [SubtitleEdit/subtitleedit#7672] [Ideas]
This is an amazing work. thanks a lot.
Could you pls add some ass functions to make it even wonderful？
1, drag the subtitle text on the video window from one pos to another,  then Set up the ass position parameters into the text automatically. instead of  {\an8} , be more precise. I know there is someway to set it, but using mouse is an easy way.
2, add some ass fade in and fade out functions.

That would make a very nice tool to handle my basic subtitle work. thanks again.

**Comment by niksedk** [comment]
SE has two ASSA plugins, see File - Plugins:
![image](https://github.com/SubtitleEdit/subtitleedit/assets/1261331/782215ec-7275-4f4e-8a2f-4f5788be8ba3)

You can also right-click on selected lines in the list view and choose some ASSA tools:
![image](https://github.com/SubtitleEdit/subtitleedit/assets/1261331/cb06ccc9-2739-4e9b-861d-684f372dbffa)


Note: Do check shortcuts for ASSA stuff/plugins:
![image](https://github.com/SubtitleEdit/subtitleedit/assets/1261331/5c5520e7-df9a-4413-822b-183a91f77ced)
